docs/src/pages/components/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -10,33 +23,118 @@ const useStyles = makeStyles({
   },
 });
 
+function escapeHTML(value) {
+  return value.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+}
+
+const EventRow = ({ event }) => {
+  const [open, setOpen] = React.useState(false);
+
+  const example = React.useMemo(() => {
+    const args = ['details // GridCallbackDetails'];
+    if (event.event) {
+      args.unshift(`event,  // ${event.event}`);
+    }
+    if (event.params) {
+      args.unshift(`params, // ${event.params}`);
+    }
+
+    return `
+const onEvent: GridEventListener<GridEvents.${event.name}> =
+ (
+    ${args.join('\n    ')}
+  ) => {...}    
+  
+// Imperative subscription    
+apiRef.current.subscribeEvent(
+  GridEvents.${event.name},
+  onEvent,
+);
+
+// Hook subscription (only available inside the scope of the grid)
+useGridApiEventHandler(GridEvents.${event.name}, onEvent);    
+`;
+  }, [event]);
+
+  return (
+    <React.Fragment key={event.name}>
+      <TableRow>
+        <TableCell style={{ borderBottom: 'unset' }}>
+          <IconButton
+            aria-label="expand row"
+            size="small"
+            onClick={() => setOpen(!open)}
+          >
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </IconButton>
+        </TableCell>
+        <TableCell style={{ borderBottom: 'unset' }}>
+          <code>{event.name}</code>
+        </TableCell>
+        <TableCell style={{ borderBottom: 'unset' }}>
+          <div
+            dangerouslySetInnerHTML={{
+              __html: escapeHTML(event.description),
```

Comment:
It's not formatting correctly.

<img width="300" src="https://user-images.githubusercontent.com/42154031/143452219-221f342d-226f-4c4a-80e5-65a712207a92.png" alt="image" style="max-width: 100%;">

Reply:
Fixed